src/js/flow/acquia_lift.variations.views.js

```diff
@@ -386,7 +381,7 @@
      */
     VariationTypeMenuListView: Backbone.View.extend({
       tagName: 'ul',
-      className: 'acquia-lift-page-variation-list',
+      className: 'acquia-lift-element-variation-list',
```

Comment:
More in the qunit tests to watch out for.  However..... those tests may be obsolete now due to the Behat tests.  They never were great but they were all we had before Behat.